### ISSUE TITLE
Basic Chemistry quest bugged

### ISSUE THREAD

**Issue by Vinc0850** [gamefreedomgit/Frostmourne#847]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/wotlk)

**Description:**
The quest Basic Chemistry in Icecrown is bugging out and is unable to be completed. Trying to add more spinal fluid to the cauldron does nothing and the attempt fails. I attempted it with a second person after several attempts and neither of us could keep the cauldron going.

**Comment by LezChap** [comment]
My understanding is it should take a minimum of 3 players to keep it going without failing...but even then we tried with a full 5 any were unable to get past the first spawning of mobs, yet never got the "Failed" message during our attempts (should say it failed the same way it tells you to add more fluid when you fail to add fluid in time).  

Might also be a cross-faction issue (two different quest IDs for Alliance and Horde, I'll post them below, maybe the cross-faction scripting is at issue, the group I tried it with was a mix of ally and horde.  Still didn't seem to matter who started or tried to continue the quest, never got past the same point.

Also, I believe slightly larger center cauldron should also work, but it didn't recognize it as a plague cauldron when we used the item next to it.

H: https://www.wowhead.com/wotlk/quest=13279/basic-chemistry
A: https://www.wowhead.com/wotlk/quest=13295/basic-chemistry
The quest item: https://www.wowhead.com/wotlk/item=44010/pustulant-spinal-fluid
